### DISCUSSION TITLE
Using different subfolders based on hostname

### DISCUSSION THREAD

**Discussion by trojkat** [twpayne/chezmoi#3303] [Q&A]
I'm migrating from a different dotfiles manager. Most of my configs are the same on different machines, in some cases, I'm adjusting them with templates but sometimes, I would like to create a different layout and use different files on different machines. I kept both layouts under the same folder and used hostname-based subfolders to choose the proper files. Example from my Polybar config:

```
dotfiles/polybar/
├── vault-13
│   ├── config
│   └── scripts
│       └── services.sh
└── vault-17
    ├── bars.ini
    ├── colors.ini
    ├── config.ini
    ├── launch.sh
    ├── modules.ini
    └── scripts
        ├── checkupdates
        ├── colors-dark.sh
        ├── colors-light.sh
        ├── color-switch.sh
        ├── launcher.sh
        ├── powermenu.sh
        ├── pywal.sh
        ├── random.sh
        ├── rofi
        │   ├── colors.rasi
        │   ├── confirm.rasi
        │   ├── launcher.rasi
        │   ├── message.rasi
        │   ├── networkmenu.rasi
        │   ├── powermenu.rasi
        │   └── styles.rasi
        └── updates.sh
```
`vault-13` and `vault-17` are the host names. How could I achieve the same with chezmoi?

**Comment by Lockszmith-GH** [comment]
This might not be what you want, but take a look at [this](https://github.com/twpayne/chezmoi/discussions/2574) and maybe [this](https://github.com/twpayne/chezmoi/discussions/2375#discussioncomment-3735286).

These are about how to manage multiple chezmoi configuration side-by-side, whether on the same git repo, or on separate ones.

Of course, others might be tackling this differently.
